devices/ITCommander.js

```diff
@@ -0,0 +1,19 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const extend = require('zigbee-herdsman-converters/lib/extend');
```

Comment:
Is not used, so you can remove this line.

Reply:
Thank You , removed

Reply:
Sorry it was not clear, just the `const extend = ...` was un used. The other should be updates like in the comment above.

Reply:
Fixed ， I am correct ?

Reply:
Close, you removed the correct line. You still need to update the other ones.

You need to replace the `zigbee-herdsman-converts/` in the require() to `../` so e.g.

```node
const tz = require('zigbee-herdsman-converters/converters/toZigbee');
```

should be

```node
const tz = require('../converters/toZigbee');
```